pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -494,7 +494,18 @@ func (pc *PodController) configurePodSecondaryNetwork(pod *corev1.Pod, networkLi
 
 	// Update the Pod's network status annotation
 	if netStatus != nil {
-		if err := netdefutils.SetNetworkStatus(pc.kubeClient, pod, netStatus); err != nil {
+		podActual, err := pc.kubeClient.CoreV1().Pods(pod.Namespace).Get(context.TODO(), pod.Name, metav1.GetOptions{})
+		if err != nil {
+			klog.ErrorS(err, "Get Pod failed", "Pod", klog.KObj(pod))
```

Comment:
"Failed to get Pod" or "Error getting Pod"

Reply:
Done.